src/components/index.ts

```diff
@@ -28,6 +28,7 @@ export { default as ListItem } from './list/ListItem';
 export { default as Menu } from './menu';
 export { default as Badge } from './badge';
 export { default as Tooltip } from './tooltip';
+export { default as Accordion } from './accordion';
```

Comment:
Do we need to export the rest of the Accordion components as well for their future use? Or do we need to add them as children in the accordion main component?

Reply:
oh, sure, good catch